winlogbeat/beater/winlogbeat.go

```diff
@@ -175,6 +175,7 @@ func (eb *Winlogbeat) Stop() {
 	logp.Info("Stopping Winlogbeat")
 	if eb.done != nil {
 		close(eb.done)
+		eb.client.Close()
```

Comment:
Yeah, something's fishy here. using 'Sync' + 'Guaranteed'. The only reason for having `ok == false` is the publisher client being closed. That is in main loop we've to break the loop on '!ok'